app/controllers/admin/users_controller.rb

```diff
@@ -50,10 +55,32 @@ def new
       @course = Course.find(params[:course_id])
     end
   end
-
+  
   # POST /admin/users/
   def create
+    @the_user = User.new(user_params)
+    @the_user.id_string = @the_user.username #workaround for redundant id_string field
```

Comment:
Perhaps a good thing for an enhancement bug fix issue later?

Reply:
@knmnyn @huyuxin0429 Yuxin is trying to improve creating users by following the Rails conventions. Let's finish up this PR and this story first. I think Yuxin can continue with this improvement at the end of this story, or create another issue and another related PR for it.

Reply:
No issues!